notebooks/samples/pytorch/lightning/TrainingAndPredictionWithPyTorchLightning.ipynb

```diff
@@ -0,0 +1,536 @@
+{
```

Comment:
<p>It's easier for readers to scan/parse the notebook overall if you take longer, detailed subtitles like these and make them shorter / more abstract. A good rule of thumb is ~5 words.</p><p> </p><p>For example, you could change this title to "Define a data processing module" and then add a sentence above the code cell to explain the details: that it's a <code>LightningDataModule</code> used to encapsulate data processing steps.</p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/GoogleCloudPlatform/ai-platform-samples/pull/241/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='16'/>